lib/Game.js

```diff
@@ -33,6 +33,25 @@ class Game {
     domUpdates.updateRoundCategory(newRoundPuzzle.currentPuzzle.category)
   }
 
+  checkLetters(letter) {
+    console.log(letter);
+    // split correct answer into an array of letters, all lowercase
+    console.log(this.round.newPuzzle.currentPuzzle.correct_answer)
+    let correctAnswer = this.round.newPuzzle.currentPuzzle.correct_answer
+    let answerLettersArray = this.round.newPuzzle.currentPuzzle.correct_answer.split('')
+    console.log(answerLettersArray)
+    // compares user inputed letter to currentPuzzle correct letters
+    // if the user guessed letter is in the currentPuzzle correct letters
+    if (answerLettersArray.includes(letter)) {
+      console.log(letter)
+      domUpdates.appendGuessToDom(letter, correctAnswer)
+      return true
+    } else {
+      // run error message
+      return false
+    }
```

Comment:
The leftover console logs and comments make the block of code pretty hard to parse.